### PR TITLE
add benchmark of workload

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**

**What this PR does / why we need it**:
add benchmark of workload
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by LiZhenCheng9527** [comment]
goos: linux
goarch: amd64
pkg: kmesh.net/kmesh/pkg/controller
cpu: Intel(R) Xeon(R) Gold 6278C CPU @ 2.60GHz
BenchmarkHandleDataWithService-4   	  141820	      8246 ns/op
PASS
ok  	kmesh.net/kmesh/pkg/controller	1.483s

File: pkg/controller/workload/workload_processor.go [hunk 1]
```diff
@@ -359,6 +359,10 @@ func (p *Processor) storeBackendData(uid uint32, ips [][]byte, portList *workloa
 	return nil
 }
 
+func (p *Processor) HandleDataWithService(workload *workloadapi.Workload) error {
```
Comment (hzxuzhonghu) [inline pkg/controller/workload/workload_processor.go:362]: we now donot need a wrapper

File: pkg/controller/workload/workload_processor_test.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+				},
+			},
+		},
+	}
+
+	workloadController := NewController(bpfLoader.GetBpfKmeshWorkload().SockConn.KmeshCgroupSockWorkloadMaps)
+
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		workload.Name = rand.String(6)
```
Comment (hzxuzhonghu) [inline pkg/controller/workload/workload_processor_test.go:78]: update  uid please

**Comment by LiZhenCheng9527** [comment]
```console
go test -bench . -benchtime 100x
goos: linux
goarch: amd64
pkg: kmesh.net/kmesh/pkg/controller/workload
cpu: Intel(R) Xeon(R) Gold 6278C CPU @ 2.60GHz
BenchmarkHandleDataWithService-4   	     100	     12300 ns/op
PASS
ok  	kmesh.net/kmesh/pkg/controller/workload	0.124s
```

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**